lib/sso_users_api/manager_job.rb

```diff
@@ -8,19 +8,24 @@ def self.delay_amount
       2
     end
 
-    def perform(id, class_name, count = 0)
+    def perform(id, class_name, on_success_callback_job = {}, count = 0)
```

Comment:
The purpose of using the hash was so that in the future, you could add other options without having to change the perform methods fingerprint. Thus, this attribute should be called "options" (or similar, and the key should be "on_success_call_back_job_name".

Reply:
Also, the hash should always be at the end of the list of attributes.

Reply:
@hoenth I've updated it as you mentioned. sorry about that.  I actually had thought about fingerprinting using named_parameters such as (id:, class_name:, count:, callbackjob:)  instead of options, but I didn't pursue it as I wasn't sure where all this gem is being used.  Using named parameters would have required updating all those places.